package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "meteor-desktop",
-  "version": "2.2.5",
+  "version": "2.2.6",
   "bin": {
     "meteor-desktop": "dist/bin/cli.js"
   },
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=4.0.0 <=12"
```

Comment:
I ran into an issue when building with node 14 but it works with node 12 so including that here. It works with Meteor 2.6 which uses node 14, so it's just the build process that doesn't work on 14

Reply:
Good for now, for the future we should probably start looking towards Node 16.